src/compiler/sys.ts

```diff
@@ -326,6 +326,9 @@ namespace ts {
                 : FileWatcherEventKind.Changed;
     }
 
+    /*@internal*/
+    export const ignoredPaths = ["/node_modules/.", "/.git"];

```

Comment:
Is there a reason we're special casing `.git` and not just ignoring all dotfolders?

Reply:
Not sure if anything starting with "." is too aggressive but we definitely have reports about ".git" and hence added that one. We can relax this further later if needed.